### PR TITLE
[improve](function) add limit check for lpad/rpad function input big value of length

### PR DESCRIPTION
**Pull request by zhangstar333**
if the user input big value of length, maybe cause some memory problem or 
exceeded size limit of thrift, so add this check to report error for user firstly.

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

File: regression-test/suites/query_p1/test_big_pad.groovy [hunk 1]
```diff
@@ -30,11 +30,16 @@ suite("test_big_pad") {
             distributed BY hash(k1) buckets 3
             properties("replication_num" = "1");
         """
-
+    test {
+        sql "select rpad('a',2000000000,'asd');"
+        exception "rpad function the length argument is 2000000000 exceeded maximum default value"
+    }
+    sql """ set repeat_max_num = 2000000001 """
```
Comment (zclllyybb) [inline regression-test/suites/query_p1/test_big_pad.groovy:37]: should add a test which could get the result after change the variable.

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run performance

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by jacktengg — APPROVED** [review]

**Pull request merged**